Worktrolly/worktrolly
# feat(Form):Modified styles and added popup in CreateNewTask, LogWork and TaskDetails

… TaskDetails

Signed-off-by: anu-aj <anugrahajawahar@gmail.com>

### **Functionality:**

- Reduce the size of Create-NewTask, Logwork, EditPage and Task-Details
- Make Logwork as pop-up
- Modify the edit page

### **Solution:**
Modified and Re-aligned the rows and columns in the above mentioned components.

### **Risk level:**

### **How to test:**

1. ng serve
2. firebase emulators:start
3. Check the Create new task page
4. Check the Task Details page
5. Click the logWork icon in Task Details page to make the logWorkPage component Pop-up
6. Click the  Edit icon in Task Details page to make the EditPage pop-up and notice the modified rows and columns in it.  

### **Screenshots:**

**-Create New Task**
![Screenshot (343)](https://user-images.githubusercontent.com/53076732/107758595-1f4d4880-6d4d-11eb-94b0-3df41959f27b.png)

**-Task Details**
![Screenshot (366)](https://user-images.githubusercontent.com/53076732/107882099-81ed4280-6f0d-11eb-9b63-d8413b60f4ea.png)

**-Task Details(After adding completion date)**
![Screenshot (378)](https://user-images.githubusercontent.com/53076732/108061644-9f8de980-707e-11eb-874a-16629555fd02.png)

**-Log Work Pop-up**
![Screenshot (424)](https://user-images.githubusercontent.com/53076732/109269613-bd9fda80-7832-11eb-8966-8fb980102e05.png)


**-Edit Page Pop-up**
![Screenshot (425)](https://user-images.githubusercontent.com/53076732/109269391-6d287d00-7832-11eb-85c6-f0cec86ac722.png)
